applications/visualizer/frontend/src/models/workspace.ts

```diff
@@ -23,7 +23,7 @@ export class Workspace {
 
     store: ReturnType<typeof configureStore>;
     layoutManager: any;
-
+    datasets: Array<Dataset>;
```

Comment:
question (non-blocking): Do we need this here?

Reply:
I guess we will need to have the datasets that are attached to a workspace for the comparison mode later no?

Reply:
I spoke with @Salam-Dalloul about this. My understanding is that the property I highlighted can be removed. Initially she though the workspace would be a good place to have the information about all datasets but after some though she decided to have them in the global context (but forgot to remove this bit).
To know what datasets are associated to a workspace we have the activeDatasets property.